Utils/XDrive.hpp

```diff
@@ -74,29 +196,23 @@ double logDriveT(double cv) { // less intense, for turning
   //return pow(fabs(cv), 1.5) / (sign(cv)*sqrt(50));
 }
 
-void autonWithXD(XDriveRobot& robot, double tgtHeading, double speed, double spinSpeed, RollingScreen& rs, double kp = 0.01) {
```

Comment:
Pls move autonWithXD to your specific project (e.g. "SpinUp"). Xdrive.hpp should ONLY hold content for general purpose XDrive, it has no knowledge of individual competition goal.

Pls bring back the simpler code of calling move() -- some simple change is needed, though. Or even better, call goStraight() when it is fully tested.